DependencyInjection/LexikTranslationExtension.php

```diff
@@ -138,9 +138,18 @@ public function prepend(ContainerBuilder $container)
             return;
         }
 
+        $rootDir = 'vendor/lexik/translation-bundle/Resources/views';
+
+        // Only symfony versions >= 3.3 include the kernel.project_dir parameter
+        if ($container->hasParameter('kernel.project_dir')) {
```

Comment:
Maybe it's better to test with `if (Kernel::VERSION_ID >= 30300) {`

Reply:
Will do, forgot that the Symfony version is present as a static kernel variable.